### PR TITLE
M0137W-660 Add section for Platform Publication

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**
Doc preview: https://docs.microej.com/en/feature-m0137w-660_platform-publication/

### CODE REVIEW

File: PlatformDeveloperGuide/platformCreation.rst [hunk 1]
```diff
@@ -322,6 +322,24 @@ the default configuration script.
 
 Modifying one of these files requires to :ref:`build the Platform <platform_build>` again.
 
+.. _platform_publication:
+
+Platform Publication
+====================
+
+The publication of the built Platform to an artifact repository is disabled by default.
```
Comment (frivieremicroej) [inline PlatformDeveloperGuide/platformCreation.rst:330]: The publication of the built Platform to a [module repository](link to module repository) is disabled by default.

**Review by frivieremicroej — CHANGES_REQUESTED** [review]
approve

File: PlatformDeveloperGuide/platformCreation.rst (hunk 1 above)
Comment (tdelhomenie-microej) [reply PlatformDeveloperGuide/platformCreation.rst]: Done

**Comment by tdelhomenie-microej** [comment]
> approve

@frivieremicroej please approve it, so I can merge it.

**Review by frivieremicroej — APPROVED** [review]

**Pull request merged**